push_value/append_list/append_list.py

```diff
@@ -0,0 +1,6 @@
+class Append_list:
```

Comment:
what is this convention camel case + snake case? You should follow a consistent naming convention. The convention says that the class name should be Camel case.

Please follow the naming convention closely and I do not need to remind you again.